packages/cosmwasm/src/signingcosmwasmclient.ts

```diff
@@ -234,6 +282,108 @@ export class SigningCosmWasmClient extends CosmWasmClient {
     };
   }
 
+  public async updateAdmin(contractAddress: string, newAdmin: string, memo = ""): Promise<ChangeAdminResult> {
+    const updateAdminMsg: MsgUpdateAdmin = {
+      type: "wasm/update-contract-admin",
+      value: {
+        sender: this.senderAddress,
+        contract: contractAddress,
+        // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
I wish there was a way to tell this rule when it's allowed and when not.

Reply:
eslint7 brings a new rule for naming conventions: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md This allows you to set: _allow strictCamelCase for variables; allow strictCamelCase or snake_case for properties_.

What I find more annoying (and this won't change) is all the needed exceptions for cases when you don't actually decide how something is named (creating objects of a type defined elsewhere, destructing objects etc.).

Reply:
Yeah, that's what I meant: what's allowed can't be expressed in terms of regexes even if the analyzer knows it's a property vs a variable.